CHANGELOG.md

```diff
@@ -22,6 +30,6 @@ include updated documentation and files outlined by the
 For more information on internal releases prior to NASA open-source approval,
 see legacy-CHANGELOG.md.
 
-[unreleased]: https://github.com/nasa/harmony-regridding-service/compare/1.0.1..HEAD
-[v1.0.1]: https://github.com/nasa/harmony-regridding-service/compare/1.0.0..1.0.1
-[v1.0.0]: https://github.com/nasa/harmony-regridding-service/compare/0.0.4-legacy..1.0.0
+[v1.0.2]: https://github.com/nasa/harmony-regridding-service/releases/tag/1.0.2
+[v1.0.1]: https://github.com/nasa/harmony-regridding-service/releases/tag/1.0.1
+[v1.0.0]: https://github.com/nasa/harmony-regridding-service/releases/tag/1.0.0
```

Comment:
@flamingbear - I think this is the transition you made in one of the recent change logs. Let me know if this is the preferred URL format you like, or if I should update to something else.